### PR TITLE
Feature signup page

### PR DESCRIPTION
**Pull request by jdemendoza**
## Description of what was implemented
-Sign up page

## Testing Notes:
Notes for any specific setup to get the branch working

------

## Checklist
- [x ] Description section in merge request filled
#### Branching
- [ x] Source branch was branched off of develop
- [x ] Target branch is merging back to develop
- [ x] Branch is correctly named
    - prefixed with `feature-`, `fix-`, `refactor-`
#### Code Quality
- [ x] Lint Passes (yarn lint)
    - `npm run lint`
- [ x] All comments are clean, concise and relevant
  * Comments are only for code blocks that need it
  * No code blocks are commented out
  * No imports are commented out
  * No debugging comments

### CODE REVIEW

File: src/index 2.css [hunk 1]
```diff
@@ -0,0 +1,13 @@
+body {
```
Comment (Dada-Tech) [inline src/index 2.css]: why is there a second index file here?

File: src/pages/.DS_Store
Comment (Dada-Tech) [inline src/pages/.DS_Store:1]: remove the .DS_Store files from your branch and update the .gitignore file to have ignore the .DS_Store file

**Review by Dada-Tech — CHANGES_REQUESTED** [review]
just a few quick changes

File: src/index 2.css (hunk 1 above)
Comment (jdemendoza) [reply src/index 2.css]: I did not notice it. Im not sure why it is in the directory. It does not seem to be imported in any other file.
Let me know if I should delete it or keep it

File: src/pages/.DS_Store
Comment (jdemendoza) [reply src/pages/.DS_Store:1]: I do not have a .DS_Store file in my local repo. I updated the .gitignore file to ignore the .DS_Store. I will go ahead and delete the file from github

**Comment by Dada-Tech** [comment]
image

its here in the project repo. you may have to:

1 temporarily remove .DS_Store from the .gitignore so your branch can track the changes
2. delete the DS_Store files in your repo
3. commit the changes (minus the gitignore)
4. Restore the gitignore file to have the .DS_Store line so it ignores other ds store files from then on

**Comment by jdemendoza** [comment]
Hi David,

I am not sure if i have done it right. I do not have any .DS_Store files in my repo, and I deleted the one on github. Let me know if I missed something

**Comment by Dada-Tech** [comment]
You can be sure by going through the files you are trying to commit here https://github.com/wydget-co/error-hub/pull/7/files
There are still DS_Store files that is trying commited to be merged.

Also, please resolve the merge conflict in your branch. There is a merge conflict in your .gitignore file. you need to 
1. pull the latest develop branch
2. merge it into your branch
3. resolve the merge conflict
4. commit and push changes

there are online guides to walk through this too for your specific setup

**Comment by jdemendoza** [comment]
I have resolved the merge conflict

File: src/index 2.css (hunk 1 above)
Comment (Dada-Tech) [reply src/index 2.css]: what purpose is it serving? It doesn't seem like its doing anything so please delete it

**Comment by Dada-Tech** [comment]
Looks good, just need to fix the navigation link in the form, and the responsive modes

**Pull request closed without merging**